examples/jsm/postprocessing/OutlinePass.js

```diff
@@ -207,11 +207,11 @@ class OutlinePass extends Pass {
 	changeVisibilityOfNonSelectedObjects( bVisible ) {
 
 		const cache = this._visibilityCache;
-		const selectedMeshes = [];
+		const selectedMeshes = new Set();
```

Comment:
Instead of creating a set per invocation of `changeVisibilityOfNonSelectedObjects()` how about transforming it to a module scope variable and clear it after each call?

Reply:
Thanks for the review!

In b073ff1 instead of turning it into module scope I converted it to an object scope variable. 
Since the meshes are calculated multiple times during the rendering I figured it would make sense to persist the result within a render call.
For that I added a `_selectedMeshesDirty` flag that is set at the start of each render call since we don't know whether the selectedObjects array has been mutated from the outside.
The selectedMeshes set could also be module scope as you suggested but i was worried that could interfere with other Outline passes if I try to persist the data between steps.
I am also happy to go for the module scope solution if its better though!

Reply:
Let's go for a module scope variable. If we are run into any issue, we can still turn it into a class member.